Could NOT find OpenSSL

Hi folks, I've been trying to install it using `yay` on Manjaro, but I'm stuck with the following error:

```
CMake Error at /usr/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:218 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY) (found
  version "1.1.1j")
Call Stack (most recent call first):
  /usr/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:582 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.19/Modules/FindOpenSSL.cmake:536 (find_package_handle_standard_args)
  deps/libimobiledevice/CMakeLists.txt:5 (find_package)
```

I've already installed `openssl` and a bunch of other things, exported the system variables pointing to open ssl directories, but still I have no clue about what I'm suppose to do to make this works.

--- a/README.md
+++ b/README.md
@@ -28,7 +28,7 @@ These instructions are tested on Ubuntu 18.04 and 20.04. For other distros/versi
 - Make sure you have the universe and multiverse repositories enabled so you will have access to FFmpeg.
 - Install OBS Studio using the [instructions on the OBS wiki](https://obsproject.com/wiki/install-instructions#ubuntu-installation).
 - Install prerequisites: `sudo apt install build-essential git cmake libavcodec-dev libssl-dev`
-  - Note: On Fedora, you will need to install both openssl-devel and openssl-static in order for OpenSSL to be detected properly.
+  - Note: On Fedora, the required package for OpenSSL is called openssl-devel.
 - Download the OBS Studio source code somewhere: 
   - `cd ~/Downloads`
   - `git clone https://github.com/obsproject/obs-studio.git`

--- a/deps/libimobiledevice/CMakeLists.txt
+++ b/deps/libimobiledevice/CMakeLists.txt
@@ -1,9 +1,7 @@
 project(libimobiledevice)
 cmake_minimum_required(VERSION 3.2)
 
-set(OPENSSL_USE_STATIC_LIBS TRUE)
 find_package(OpenSSL REQUIRED)
-set(OPENSSL_USE_STATIC_LIBS TRUE)
 
 message("OpenSSL include dir: ${OPENSSL_INCLUDE_DIR}")
 message("OpenSSL libraries: ${OPENSSL_LIBRARIES}")

